unipipe/commands/list.ts

```diff
@@ -137,3 +159,19 @@ function profileColValues(
     }
   }
 }
+
+function buildFilterFn(opts: ListOpts ): (instance: ServiceInstance) => Promise<boolean> {
+  return ( async (instance: ServiceInstance) => {
+    
+    var statusFilterMatches = true
+    if (opts.status !== undefined && opts.status !== null) {
```

Comment:
d: I think you can just simplify this to a single boolean expression, which would make this more readable

Reply:
like `statusFilterMatches = ...` and  `deletedFilterMatches = :..`